apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Tests/KeysTests.cs

```diff
@@ -0,0 +1,177 @@
+// Copyright 2021 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Cloud.Spanner.V1;
+using Google.Protobuf.WellKnownTypes;
+using System;
+using Xunit;
+
+namespace Google.Cloud.Spanner.Data.Tests
+{
+    public class KeysTests
+    {
+        [Fact]
+        public void CreateKeyFromParameterCollection()
+        {
+            var key = new Key(
+                new SpannerParameterCollection
+                {
+                    { "", SpannerDbType.Bool, true },
```

Comment:
Keys only contain the values and types, but no (column) names, so when a `SpannerParameterCollection` is used, the name of the parameter is not used.